src/core/Database.cpp

```diff
@@ -32,6 +32,8 @@
 #include <QTemporaryFile>
 #include <QTimer>
 
+#include <iostream>
```

Comment:
What is this for?

Reply:
My bad. That was for testing, it shouldn't be there. I will remove it.